### ISSUE TITLE
*feature request* add restaurants

### ISSUE THREAD

**Issue by LucasPuente** [underthecurve/bourdain-travel-places#1]
Would be really cool to replicate this with a list of restaurants Bourdain ate at.

**Comment by tsp2123** [comment]
Is that data published somewhere—are there logs on CNN's websites? I'd love
to scrape for it.

**Comment by underthecurve** [comment]
couldn't find the complete logs on the CNN websites for restaurants he traveled to ... this is good for a few locations: https://explorepartsunknown.com/ keep in mind that's just Parts Unknown though.

**Comment by LucasPuente** [comment]
Yeah, CNN has published the transcripts of _Parts Unknown_ here: http://transcripts.cnn.com/TRANSCRIPTS/abpu.html.

And it looks like there are (at least some) transcripts from _No Reservations_ here on a site called tv.ark.com: https://www.google.com/search?biw=2560&bih=1343&ei=6vlQW9i3Foit0gKT3oVY&q=%22Anthony+Bourdain%3A+No+Reservations%22+site%3Atv.ark.com&oq=%22Anthony+Bourdain%3A+No+Reservations%22+site%3Atv.ark.com&gs_l=psy-ab.3...24241.26238.0.26430.15.15.0.0.0.0.148.1173.12j3.15.0..2..0...1.1.64.psy-ab..0.5.468...0j0i22i30k1j33i21k1j33i160k1.0.eE4zbQ9VH2g.
